pystan/__init__.py

```diff
@@ -17,6 +18,13 @@
 if len(logger.handlers) == 1:
     logging.basicConfig(level=logging.INFO)
 
+if platform.system() == "Windows":
+    add_libtbb_path()
+
+# clean imports
```

Comment:
This practice of deleting imports is new to me. Is this something other libraries do? What's the justification?

edit: nevermind, I didn't realize this was in `__init__.py`. Why does add_libtbb_path need to be added here and not in models.py?

Reply:
It is a runtime dependency, so it should be defined at runtime.

Calling it in model.sampling etc could work, but this way it is always defined when we import the lib.